src/js/messaging/utils/helpers.js

```diff
@@ -85,7 +85,7 @@ export function formattedDate(date, options = {}) {
   let dateString;
 
   if (momentDate.isSame(now, 'd')) {
-    dateString = momentDate.format('HH:mm');
+    dateString = `${momentDate.format('HH:mm')} <abbr title="Eastern Standard Time">EST</abbr>`;
```

Comment:
I think this literally prints out the full string un-marked-up. To return it as markup, it should be:

`<span>{momentDate.format('HH:mm')} <abbr title="Eastern Standard Time">EST</abbr></span>`

And also import React in this module.